code/drasil-data/Data/Drasil/Concepts/Math.hs

```diff
@@ -25,6 +25,15 @@ angle, area, axis, calculation, cartesian, centre, change, constraint, diameter,
   rightHand, shape, surArea, surface, unitV, unit_, vector, xAxis, xCoord, xComp, xDir,
   yAxis, yCoord,  yComp, yDir, zAxis, zCoord, zComp, zDir, iAngle :: ConceptChunk
 
+symbols :: [QuantityDict]
+symbols = [qdNegInf, qdPosInf, qdInvLaplaceTransform] 
+qdNegInf, qdPosInf, qdInvLaplaceTransform :: QuantityDict
+
+symnegInf, symposInf, syminvLaplace :: Symbol
+symnegInf = Variable "-∞"
+symposInf = Variable "∞"
+syminvLaplace = Variable "L⁻¹[F(s)]"
```

Comment:
Hmm, this is a hack as a 'symbol', so let's not put that in `Concepts.Math` right now.